+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Tue May 30 16:03:25 2023 - INFO  - Start environmental preparation.
+ touch test.txt
+ DNF_INSTALL 'cobbler httpd'
+ pkgs='cobbler httpd'
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs 'cobbler httpd' --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp930tr68i
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp930tr68i
+ systemctl start httpd
+ systemctl start cobblerd
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Tue May 30 16:07:55 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run testcase.'
+ message='Start to run testcase.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run testcase.'
Tue May 30 16:07:58 2023 - INFO  - Start to run testcase.
+ cobbler file add --name=OpenEuler1 --path=test.txt --owner=root --group=root --mode=mode --template=tem
cobblerd does not appear to be running/accessible: ConnectionRefusedError(111, 'Connection refused')
Traceback (most recent call last):
  File "/usr/bin/cobbler", line 35, in <module>
    sys.exit(app.main())
  File "/usr/lib/python3.10/site-packages/cobbler/cli.py", line 852, in main
    rc = cli.run(sys.argv)
  File "/usr/lib/python3.10/site-packages/cobbler/cli.py", line 418, in run
    self.token = self.remote.login("", self.shared_secret)
  File "/usr/lib64/python3.10/xmlrpc/client.py", line 1122, in __call__
    return self.__send(self.__name, args)
  File "/usr/lib64/python3.10/xmlrpc/client.py", line 1464, in __request
    response = self.__transport.request(
  File "/usr/lib64/python3.10/xmlrpc/client.py", line 1166, in request
    return self.single_request(host, handler, request_body, verbose)
  File "/usr/lib64/python3.10/xmlrpc/client.py", line 1196, in single_request
    raise ProtocolError(
xmlrpc.client.ProtocolError: <ProtocolError for 127.0.0.1:80/cobbler_api: 503 Service Unavailable>
+ CHECK_RESULT 1 0 0 'Failed option: file add'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: file add'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: file add'
+ LOG_ERROR 'Failed option: file add'
+ message='Failed option: file add'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed option: file add'
Tue May 30 16:08:17 2023 - ERROR - Failed option: file add
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_cobbler_file.sh line 34'
+ message='oe_test_cobbler_file.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_cobbler_file.sh line 34'
Tue May 30 16:08:19 2023 - ERROR - oe_test_cobbler_file.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ cobbler file report --name=OpenEuler1
+ grep 'Path.*: test.txt'
+ CHECK_RESULT 1 0 0 'Failed option: file add'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: file add'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: file add'
+ LOG_ERROR 'Failed option: file add'
+ message='Failed option: file add'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed option: file add'
Tue May 30 16:08:36 2023 - ERROR - Failed option: file add
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_cobbler_file.sh line 36'
+ message='oe_test_cobbler_file.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_cobbler_file.sh line 36'
Tue May 30 16:08:38 2023 - ERROR - oe_test_cobbler_file.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ cobbler file copy --name=OpenEuler1 --newname=OpenEuler2
exception on server: 'internal error, unknown file name OpenEuler1'
+ CHECK_RESULT 0 0 0 'Failed option: file copy'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: file copy'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ cobbler file list
+ grep OpenEuler2
+ CHECK_RESULT 1 0 0 'Failed option: file copy'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: file copy'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: file copy'
+ LOG_ERROR 'Failed option: file copy'
+ message='Failed option: file copy'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed option: file copy'
Tue May 30 16:09:07 2023 - ERROR - Failed option: file copy
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_cobbler_file.sh line 40'
+ message='oe_test_cobbler_file.sh line 40'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_cobbler_file.sh line 40'
Tue May 30 16:09:09 2023 - ERROR - oe_test_cobbler_file.sh line 40
+ '[' 0 -eq 1 ']'
+ return 0
+ cobbler file edit --name=openEuler1 --owners=tom
exception on server: 'internal error, unknown file name openEuler1'
+ CHECK_RESULT 0 0 0 'Failed option: file edit'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: file edit'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ grep tom
+ cobbler file report --name=openEuler1
+ CHECK_RESULT 1 0 0 'Failed option: file edit'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: file edit'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: file edit'
+ LOG_ERROR 'Failed option: file edit'
+ message='Failed option: file edit'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed option: file edit'
Tue May 30 16:09:38 2023 - ERROR - Failed option: file edit
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_cobbler_file.sh line 44'
+ message='oe_test_cobbler_file.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_cobbler_file.sh line 44'
Tue May 30 16:09:40 2023 - ERROR - oe_test_cobbler_file.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ cobbler file list
+ grep OpenEuler
+ CHECK_RESULT 1 0 0 'Failed option: file list'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: file list'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: file list'
+ LOG_ERROR 'Failed option: file list'
+ message='Failed option: file list'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed option: file list'
Tue May 30 16:09:56 2023 - ERROR - Failed option: file list
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_cobbler_file.sh line 46'
+ message='oe_test_cobbler_file.sh line 46'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_cobbler_file.sh line 46'
Tue May 30 16:09:57 2023 - ERROR - oe_test_cobbler_file.sh line 46
+ '[' 0 -eq 1 ']'
+ return 0
+ cobbler file find
+ grep OpenEuler
+ CHECK_RESULT 1 0 0 'Failed option: file find'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: file find'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: file find'
+ LOG_ERROR 'Failed option: file find'
+ message='Failed option: file find'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed option: file find'
Tue May 30 16:10:13 2023 - ERROR - Failed option: file find
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_cobbler_file.sh line 48'
+ message='oe_test_cobbler_file.sh line 48'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_cobbler_file.sh line 48'
Tue May 30 16:10:15 2023 - ERROR - oe_test_cobbler_file.sh line 48
+ '[' 0 -eq 1 ']'
+ return 0
+ cobbler file remove --name=OpenEuler1
exception on server: 'internal error, unknown file name OpenEuler1'
+ CHECK_RESULT 0 0 0 'Failed option: file remove'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: file remove'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ cobbler file list
+ grep OpenEuler1
+ CHECK_RESULT 1 0 1 'Failed option: file remove'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='Failed option: file remove'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ cobbler file rename --name=OpenEuler2 --newname=OpenEuler3
exception on server: 'internal error, unknown file name OpenEuler2'
+ CHECK_RESULT 0 0 0 'Failed option: file rename'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: file rename'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ cobbler file list
+ grep OpenEuler3
+ CHECK_RESULT 1 0 0 'Failed option: file rename'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: file rename'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: file rename'
+ LOG_ERROR 'Failed option: file rename'
+ message='Failed option: file rename'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed option: file rename'
Tue May 30 16:11:13 2023 - ERROR - Failed option: file rename
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_cobbler_file.sh line 56'
+ message='oe_test_cobbler_file.sh line 56'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_cobbler_file.sh line 56'
Tue May 30 16:11:15 2023 - ERROR - oe_test_cobbler_file.sh line 56
+ '[' 0 -eq 1 ']'
+ return 0
+ cobbler file report --name=openEuler3
+ grep Name
+ CHECK_RESULT 1 0 0 'Failed option: distro report'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: distro report'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: distro report'
+ LOG_ERROR 'Failed option: distro report'
+ message='Failed option: distro report'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed option: distro report'
Tue May 30 16:11:30 2023 - ERROR - Failed option: distro report
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_cobbler_file.sh line 58'
+ message='oe_test_cobbler_file.sh line 58'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_cobbler_file.sh line 58'
Tue May 30 16:11:32 2023 - ERROR - oe_test_cobbler_file.sh line 58
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run testcase.'
+ message='End to run testcase.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run testcase.'
Tue May 30 16:11:34 2023 - INFO  - End to run testcase.
+ CASE_RESULT 0
+ case_re=0
+ test -z 8
+ test 8 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue May 30 16:11:36 2023 - ERROR - failed to execute the case.
+ exit 8
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Tue May 30 16:11:39 2023 - INFO  - Start to restore the test environment.
+ rm -rf test.txt
+ systemctl stop httpd
+ systemctl stop cobblerd
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp930tr68i ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp930tr68i
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Tue May 30 16:13:59 2023 - INFO  - End to restore the test environment.
